invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/macros/files.html

```diff
@@ -69,7 +69,12 @@
           </div>
           </small>
         </td>
-        <td>{{ file.size|filesizeformat }}</td>
+        <td>
+          {%if file.size %} 
```

Comment:
@slint shouldn't always `size` be defined if there is an uploaded file? I recall that the size attribute is missing if an upload is in progress but after that it should be ok, no?